07/homework.py

```diff
@@ -0,0 +1,29 @@
+phone_book = {}
+while True:
+    command = input('Enter a command: ')
+    if command == 'stats':
+        print(f'Number of records is {len(phone_book.keys())}')
+    elif command == 'add':
+        name = input('Enter a name: ')
+        phone_number = input('Enter a phone number: ')
+        print(f'{name} is added to a phone book')
+        phone_book[name] = phone_number
```

Comment:
Please, check before adding if such a case already exists. The task does not allow rewriting records, only deleting and adding new ones.